Cargo.toml

```
@@ -18,7 +18,7 @@ license = "MIT"
 readme = "README.md"
 
 [dependencies]
-serde = { version = "1.0.144", optional = true }
+serde = { version = "1.0.144", optional = true, default-features = false }
```

@AverageHelper Thank you for your PR. It seems the `alloc` feature is missing for `serde`.

```suggestion
serde = { version = "1.0.144", optional = true, default-features = false, features = ["alloc"] }
```